tree/dataframe/inc/ROOT/RSnapshotOptions.hxx

```diff
@@ -11,8 +11,9 @@
 #ifndef ROOT_RSNAPSHOTOPTIONS
 #define ROOT_RSNAPSHOTOPTIONS
 
+#include "ROOT/RNTupleWriteOptions.hxx"
```

Comment:
Apologies for bringing this up only now. By looking at the code more closely, I am reminded that a lot of work has been put into removing all header inclusions in RDataFrame of other parts of ROOT I/O (namely TTree, TChain, TFile and related headers). This now introduces another header inclusion which is logically similar to including something from TTree, and I'm afraid it goes against the rest of the work that was done. I don't have yet a clear idea on how to overcome this.